_drafts/SC3-1-1-html-lang-valid.md

```diff
@@ -0,0 +1,44 @@
+---
+name: SC3-1-1-html-lang-valid
+
+description:
+ This rule checks the `xml:lang` and/or the `lang` attribute, and that it conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
+
+success_criterion:
+- 3.1.1
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Annika Nietzio
+- Jey Nandakumar
+---
+
+## Test Procedure
+
+### Applicability
+
+The root node of the page, if it is an `html` element with a `lang` and/or `xml:lang` attribute.
```

Comment:
The root node of the page will always be a `Document`; you likely mean root element instead.

Reply:
Cool, changed to root element.

Reply:
@kasperisager 
done - review/ approve please.